contracts-v0.2/deploy/Aggregator.s.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
```

Comment:
Since we are planning to redeploy all `Aggregator`s and `SubmissionProxy`, maybe we should update the version of Solidity. What do you think?